lonelyplanet/bane
# AMD path auto builder for Webpack

Added AMD path auto builder. Added an `enabled` property to grunt-notify notification options so that teammates can choose which notifications they want to see.

## Documentation update

Some general updates to reflect current state of development, and also added a section about Doodads.

## AMD path

I added a `main.js` file to each of the component directories that were missing it. With the `amd_path_builder.js` module, it now includes every folder that exists in `ui/components` as an entry path.

## Notifications

To enable notification settings, perform the following steps

1. `cp grunt/notify.js.example grunt/notify.js`
1. Change the **enabled** property value to `true` in notify.js

## Verification tasks

1. Pull branch
1. `cp grunt/notify.js.example grunt/notify.js`
1. Run `grunt`
1. Verify that there are no failures in compiling component directories
1. Verify that you **do not** get a system notification when build succeeds
1. Change the **enabled** property value to `true` in notify.js
1. Run `grunt` again
1. Verify that you **do** get the system notification
